src/main/kotlin/com/example/mykku/member/domain/Member.kt

```
@@ -0,0 +1,29 @@
+    val followerCount: Int = 0,
+
+    @Column(name = "following_count")
+    val followingCount: Int = 0,
+
+    @OneToMany(mappedBy = "member")
+    val saveFeeds: List<SaveFeed> = mutableListOf(),
+
+    @OneToMany(mappedBy = "member")
+    val saveDailyMessages: List<SaveDailyMessage> = mutableListOf(),
+) {
+}
```

detekt (1.23.7)

[warning] The class or object Member is empty.

(detekt.empty-blocks.EmptyClassBlock)